core/src/com/unciv/ui/utils/ImageGetter.kt

```diff
@@ -256,6 +258,63 @@ object ImageGetter {
         return advancementGroup
     }
 
+    fun getInfluenceBar(influence: Float, relationshipLevel: RelationshipLevel, influenceBarSize: Float = 100f): Table {
```

Comment:
This is *only* used by the city button, so it should be a function in the city button